src/js/components/RegisterPage/SignUp.js.orig

```diff
@@ -0,0 +1,173 @@
+import React, { Component } from "react";
```

Comment:
remove .orig files 
and it to gitignore

Reply:
Done and Added to git

Reply:
Added to .gitignore